src/example19.py

```diff
@@ -0,0 +1,25 @@
+# 19 XML/HTML parsing
+
+import xml.etree.ElementTree as etree
+
```

Comment:
1 line between import and code (2 lines in another task). it should be consistent